src/graph_notebook/seed/queries/propertygraph/identity/00_identity_group_nodes_gremlin.txt

```diff
@@ -0,0 +1,3 @@
ddV("identityGroup").property(id,"0005815c811baa6b502e061f457f8a70389cab52").property("igid","0005815c811baa6b502e061f457f8a70389cab52").property("type","household").addV("identityGroup").property(id,"0009b73d8484165f83059dbb90467ae40367921a").property("igid","0009b73d8484165f83059dbb90467ae40367921a").property("type","household").addV("identityGroup").property(id,"000c2e874a5eef24b0a9de12c6ba292bae1eb84b").property("igid","000c2e874a5eef24b0a9de12c6ba292bae1eb84b").property("type","company").addV("identityGroup").property(id,"000c2f6447a030125fa588b74cc5bddab22c2879").property("igid","000c2f6447a030125fa588b74cc5bddab22c2879").property("type","household").addV("identityGroup").property(id,"000d7a2fc1f89e2dbafa8cac37663c6c2d483a64").property("igid","000d7a2fc1f89e2dbafa8cac37663c6c2d483a64").property("type","household").addV("identityGroup").property(id,"000e3996595fb30778cfa8d76cd77a4daae360d7").property("igid","000e3996595fb30778cfa8d76cd77a4daae360d7").property("type","household").addV("identityGroup").property(id,"0018fa95475264aa11104519c12beed8149c791f").property("igid","0018fa95475264aa11104519c12beed8149c791f").property("type","company").addV("identityGroup").property(id,"002118a454ea266dc4a9d11f16eabd7380acc2ac").property("igid","002118a454ea266dc4a9d11f16eabd7380acc2ac").property("type","household").addV("identityGroup").property(id,"00239983a4438c12aa59757ba42c99572f8357ed").property("igid","00239983a4438c12aa59757ba42c99572f8357ed").property("type","household").addV("identityGroup").property(id,"0028736be158de05e73cc364e0def636a1b01dd9").property("igid","0028736be158de05e73cc364e0def636a1b01dd9").property("type","household").addV("identityGroup").property(id,"0030b4b9be160f3b7684ca631a37eb6d8d0ceed1").property("igid","0030b4b9be160f3b7684ca631a37eb6d8d0ceed1").property("type","household").addV("identityGroup").property(id,"003bea55c68c7625555c43c3375b141fd154210f").property("igid","003bea55c68c7625555c43c3375b141fd154210f").property("type","household").addV("identityGroup").property(id,"003d7adb821efac06d6de7aef472175233d0c1fa").property("igid","003d7adb821efac06d6de7aef472175233d0c1fa").property("type","household").addV("identityGroup").property(id,"003d906d79e3534d7c3ca7ff58a0205186255d3b").property("igid","003d906d79e3534d7c3ca7ff58a0205186255d3b").property("type","household").addV("identityGroup").property(id,"004090ec584eb447086e5370a442da6e9538820e").property("igid","004090ec584eb447086e5370a442da6e9538820e").property("type","household").addV("identityGroup").property(id,"0044ce3eb555ccb356eaf83040eec2f17fb435e7").property("igid","0044ce3eb555ccb356eaf83040eec2f17fb435e7").property("type","household").addV("identityGroup").property(id,"0047bdcec10d7329fda1efc3af6f798b0fa1291f").property("igid","0047bdcec10d7329fda1efc3af6f798b0fa1291f").property("type","household").addV("identityGroup").property(id,"0048499bcd5b698d92a9d0f95669f05048227c5e").property("igid","0048499bcd5b698d92a9d0f95669f05048227c5e").property("type","household")
```

Comment:
I don't think these files are supposed to be included in the package due to size

Reply:
Removed